### DISCUSSION TITLE
Any plans to have a Set type?

### DISCUSSION THREAD

**Discussion by avrame** [vlang/v#19542] [Questions and Answers]
I'm building a simple web browser and it would be nice to implement the "class" attribute as a Set so duplicate classes are ignored. Also, I think something like Rust's HashSet can determine if an item is in the Set in constant time whereas using an Array is O(n), right?

**Comment by medvednikov** [comment]
Hi

`datatypes.Set`

<img width="412" alt="image" src="https://github.com/vlang/v/assets/687996/0f5b0d8b-01a5-4bea-b28b-5c282d77c9fc">

**Reply by avrame** [reply]
Oh nice! I didn't see this in the documentation - does it need to be added?

**Reply by JalonSolov** [reply]
https://modules.vlang.io/datatypes.html
